.github/workflows/release-testing.yaml

```diff
@@ -0,0 +1,125 @@
+name: release testing artifacts
+
+on:
+  push:
+    branches:
+      - testing
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  build-container-image:
+    strategy:
+      matrix:
+        # TODO: Add arm64 runner
```

Comment:
need a separate runner because we need to generate and compile cpp code, that's the reason, right? (can't cross-compile)